apps/pm-user/src/pages/login/Login.js

```diff
@@ -0,0 +1,49 @@
+import { useOktaAuth } from '@okta/okta-react';
+import { LoginForm } from '../../components';
+import styles from './login.module.scss';
+
+const Login = () => {
+  const { oktaAuth, authState } = useOktaAuth();
+
+  const login = async (password, email) =>
```

Comment:
Renaming this function to "loginHandler" feels appropriate, as it is being passed into LoginForm component.

Reply:
Issue resolved.